Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelTests.swift

```diff
@@ -1,28 +1,51 @@
+import NetworkService
 import XCTest
 
 final class SitterProfileViewModelTests: XCTestCase {
+    override func setUp() {
+        super.setUp()
+        URLProtocol.registerClass(MockURLProtocol.self)
+    }
+
+    override func tearDown() {
+        URLProtocol.unregisterClass(MockURLProtocol.self)
+        super.tearDown()
+    }
+
     func testSitterProfileViewModelCanBeInitialized() {
         _ = SitterProfileViewModel()
     }
 
-    func testSitterProfileViewModelInitializedWithLastSavedData() {
+    func testSitterProfileViewModelInitializedWithLastSavedData() async {
```

Comment:
What is the difference between this test and `testSaveMethodSuccessfullySavedChangedSitterProperties` in the `SitterProfileViewModelSaveTests` class?

Reply:
You're right, `testSitterProfileViewModelInitializedWithLastSavedData()` deleted with e9d20b2